### PR TITLE
u-boot-fio recipe rework

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool_%.bbappend [hunk 1]
```diff
@@ -1,45 +1,6 @@
...
-                if [ $type_idx -eq $machine_idx ]; then
-                    install -d ${DEPLOYDIR}/${BOOT_TOOLS}
-                    # When sign is enabled the final DTB should be copied from deploy dir
-                    if [ "${UBOOT_SIGN_ENABLE}" = "1" ]; then
-                        install -m 0644 ${DEPLOY_DIR_IMAGE}/${UBOOT_DTB_IMAGE} ${DEPLOYDIR}/${BOOT_TOOLS}/${UBOOT_DTB_NAME}
-                    else
-                        install -m 0644 ${B}/${config}/arch/arm/dts/${UBOOT_DTB_NAME} ${DEPLOYDIR}/${BOOT_TOOLS}
-                    fi
-                    install -m 0644 ${B}/${config}/spl/u-boot-spl-nodtb.bin ${DEPLOYDIR}/${BOOT_TOOLS}/u-boot-spl-nodtb.bin-${MACHINE}-${UBOOT_CONFIG}
-                    install -m 0644 ${B}/${config}/u-boot-nodtb.bin ${DEPLOYDIR}/${BOOT_TOOLS}/u-boot-nodtb.bin-${MACHINE}-${UBOOT_CONFIG}
```
Comment (igoropaniuk) [inline meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool_%.bbappend:22]: the only difference between `u-boot-fio-mfgtool_%.bbappend` and `u-boot-fio_%.bbappend` was `s/${UBOOT_CONFIG}/${type}/g`, not sure if I can use `${type}` for both

@ricardosalveti

Comment (ricardosalveti) [reply meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool_%.bbappend:22]: Should be fine to have type for both as it iterates over UBOOT_CONFIG, and in our case we usually have only one config.

**Review by quaresmajose — APPROVED** [review]
LGTM

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by angolini — APPROVED** [review]

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**